addons/mail/static/src/new/web/chatter.xml

```diff
@@ -77,7 +77,7 @@
         </div>
         <t t-if="thread.composer.type">
             <t t-if="thread.composer.type === 'message'">
-                <small class="px-3 pb-1 pt-2" style="margin-left:48px;"><em class="text-muted">To followers of:</em> <b class="px-1">"<t t-esc="props.displayName"/>"</b></small>
+                <small class="px-3 pb-1 pt-2" style="margin-left:48px;"><em class="text-muted">To followers of:</em> <b class="px-1">"<t t-esc="thread.displayName"/>"</b></small>
```

Comment:
can we unskip display name test with this change?

Reply:
The test still doesn't pass